languages/rust/exercises/concept/entry-api/.docs/instructions.md

```diff
@@ -0,0 +1,24 @@
+# Instructions
+
+In this exercise you'll be using a `HashMap`, along with entry API methods, to solve a simple algorithm problem.
+
+The problem states: given a slice of `&str`s representing a magazine and another slice of `&str`s representing a note you're looking to construct, implement a function that returns a `bool` indicating whether the note can be constructed from the magazine. In other words, does the magazine contain at least as many instances of each word required by the note?
+
+You'll start with the following stubbed function signature:
+
+```rust
+pub fn construct_note(magazine: &[&str], note: &[&str]) -> bool {
+
+}
+```
+
+Given the following input
+
+```rust
+let magazine = &["two", "times", "three", "is", "not", "four"];
+let note = &["two", "times", "two", "is", "four"];
+```
+
+the function should return `false` since the magazine only contains one instance of `"two"` when the note requires two of them.
+
+Note that case sensitivity matters.
```

Comment:
```suggestion
    ```rust
    let magazine = "two times three is not four".split_whitespace();
    let note = "two times two is four".split_whitespace();
    assert_eq!(can_construct_note(magazine, note), false);
    ```
    
    The function returns `false` since the magazine only contains one instance of `"two"` when the note requires two of them.
```

This construction is shorter and more explicit. The note about capitalization has been moved to a previous section.

If doing this manually, please omit the leading spaces; those were only necessary to get github's suggestion box to work properly.